content/doc/developer/publishing/wiki-page.adoc

```diff
@@ -52,33 +54,59 @@ Since Jenkins will link to the plugin's page on the plugin site from its plugin
 
 == Migrating from Wiki to GitHub
 
-Document migration can be done in a semi-automated way using the link:https://pandoc.org[Pandoc] tool for Wiki->Asciidoc conversion.
+Document migration can be done in a semi-automated way using the link:https://pandoc.org[Pandoc] tool for Wiki conversion to Markdown or Asciidoc.
 This guide shows how to do so for a common case when the documentation page is going to be sourced from the README file inside the repository's root.
 
 NOTE: If there is no ticket for the documentation migration created by the current plugin maintainers,
 make sure to create one and then discuss with the plugin maintainers.
 Similarly, Asciidoc/Markdown preferences should be also discussed with maintainers.
 
+There are the following migration steps:
+
 . Fork the plugin repository in GitHub and clone it to the local machine.
+. Export the documentation. It can be done in a manual or automated way, see below
+. Add the documentation and images to the repository
+** You can merge the exported file with existing README file or create a new one.
+. Copy-edit the documentation, see below
+. Modify the URL documentation page reference in the project file so that it points to GitHub (link:/doc/developer/publishing/documentation/#referencing-the-documentation-page-from-the-project-file[documentation]).
+. Commit changes, push them to your fork and create a pull request against the repository.
+
+=== Exporting Wiki documentation (automatic way)
+
+It is possible to use link:https://jenkins-wiki-exporter.g4v.dev/[Jenkins Wiki Exporter] in order to export plugin documentation to Markdown or Asciidoc.
+
+. Go to the service
+. Specify the plugin ID you want to export in the field
```

Comment:
When I enter `git` or `git-client`, `platformlabeler`, `label-verifier (all ID's of valid plugins that have wiki pages) into the page, it reports `Internal server error` or reports `Not a wiki page`.

Can you help me understand the mistake I'm making so that others can avoid that mistake?

Reply:
https://github.com/halkeye/jenkins-wiki-exporter/issues/4